SimplyPluralApiCommons.yml

```diff
@@ -26,12 +26,14 @@ components:
           description: Whether the description supports markdown
         timestamp:
           description: Milliseconds since epoch for this timestamp/date
+        lastOperationTime:
+          description: Time in milliseconds since epoch of the last operation on this object
```

Comment:
"last update on this object" is more specific, operation could be anything, but it's only when updating or setting. I know the variable is then misnamed, but at least by writing specifically it is because of an update, it's more clear what it is.